docs/cimpl6LanguageReference.md

```diff
@@ -1057,124 +1044,127 @@ With FHIR as the target output, the right-hand side of the map statement can poi
 
-See the FHIR documentation for more detail about [slicing](https://www.hl7.org/fhir/profiling.html#slicing).
+In CIMPL, the [`includes`](#includes-constraint) constraint creates slices. This section discusses how slices defined in the Class file are mapped to FHIR.
 
-#### Slicing Rules
 <!-- Carmela - add text to relate to the class file slicing definition - they are related -->
-Slicing is used to differentiate occurrences of an `Element`, modeling it as a sub-list. An example of this is the [**FHIR Blood Pressure profile**](https://www.hl7.org/fhir/profiling.html#slicing), which is an **Observation** with two **Components**, identical in structure other than their represented code.
 
-In CIMPL, slicing rules are specified in the Map file.  Four reserved phrases are used in conjunction to specify the slice:
+Four reserved phrases are used to specify the mapping of the slices:
 
 * `slice on` (required)
 * `slice on type` (optional)
 * `slice strategy` (optional)
 * `slice at` (optional)
 
-#### Declare a Discriminator
-
-Discriminators are used to distinguished slices. FHIR requires discriminators to uniquely identify the slice to which an instance of data belongs.
+#### Discriminator Path: `slice on`
 
-##### Slice On
-<!-- MK:  the second sentence needs work, but I don't know what to change it to.  Also - the example (based on the 2nd part = coding) seems to be referencing a FHIR data type, and if the slice is on a FHIR property (includes) - I'm missing the connection. Suggest an example that doesn't have the same word repeating.  Need help on this paragraph and example in #### Slice Strategy   New topic -  The 5# headings for slice on and discriminator type don't stand out in the rendered view  -->
-
-Declare a discriminator path to perform slicing, using `slice on` followed by the discriminator path. This is the path to the `Property` whose data identifies the `slice on` instance belongs to. For example, if `slice on =` _code.coding.code_, the code in each slice must be a unique, fixed value, and every instance must have a code that matches one of the defined slices.
+FHIR requires discriminators to uniquely identify the slice to which an instance of data belongs. The path to the discriminator is specified in the `slice on` parameter. For example, if `slice on = code.coding.code`, the code in each slice must be a unique, fixed value, and every instance must have a code that matches one of the defined slices.
 
 For more explanation of how discriminators are defined in FHIR, see [discriminators](https://www.hl7.org/fhir/profiling.html#discriminator).
 
-##### Discriminator Type
+#### Discriminator Type: `slice on type`
```

Comment:
Is there another discriminator type - based on the levels of headings, - the only one is slice on type - so either the heading should be Slice on Type, or Discriminator Type.

Reply:
There are four parameters:
* `slice on` (required) = Discriminator
* `slice on type` (optional) = Discriminator Type
* `slice strategy` (optional) = Slice Strategy
* `slice at` (optional) = Slice Location
They are (and should be) all at the same level.